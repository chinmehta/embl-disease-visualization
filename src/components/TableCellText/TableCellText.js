import React from "react";

function TableCellText({ text }) {
  return (
    <>
      <td className="px-2 border-collapse border border-gray-400">{text}</td>
    </>
  );
}

export default TableCellText;
