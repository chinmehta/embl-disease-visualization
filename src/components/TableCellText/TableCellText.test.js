import React from "react";
import renderer from "react-test-renderer";

import TableCellText from "./TableCellText";

it("renders when no text is passed as prop", () => {
  const element = renderer.create(<TableCellText />).toJSON();

  expect(element).toMatchSnapshot();
});

it("renders when text is passed as prop", () => {
  const element = renderer.create(<TableCellText text="0.987" />).toJSON();

  expect(element).toMatchSnapshot();
});
