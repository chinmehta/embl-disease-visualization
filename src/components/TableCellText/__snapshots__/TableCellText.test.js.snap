// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when no text is passed as prop 1`] = `
<td
  className="px-2 border-collapse border border-gray-400"
/>
`;

exports[`renders when text is passed as prop 1`] = `
<td
  className="px-2 border-collapse border border-gray-400"
>
  0.987
</td>
`;
